src/execution_plan/optimizations/seek_by_id.c

```diff
@@ -48,102 +49,80 @@ static bool _idFilter(FT_FilterNode *f, int *rel, EntityID *id, bool *reverse) {
 	return true;
 }
 
-static void _setupIdRange(int rel, EntityID id, bool reverse, NodeID *minId, NodeID *maxId,
-						  bool *inclusiveMin, bool *inclusiveMax) {
-	switch(rel) {
-	case OP_GT:
-		*minId = id;
-		break;
+static void _reverseOp(AST_Operator *op) {
```

Comment:
isn't this code a duplicate of `_reverseOp` from utilize_indices.c
can we find a way to share this function?

Reply:
I wrote it myself as I was unaware of this function. created `optimizations_shared.h`